manageiq-operator/cmd/manager/main.go

```diff
@@ -29,6 +29,7 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+	routev1 "github.com/openshift/api/route/v1"
```

Comment:
Will this prevent us from running on plain Kubernetes?  If so, I don't think we can use this.

Reply:
Yes, this part is OpenShift only.  It's conditional, so we only try to create a route if we have a certificate for the ingress.  The default ingress can not handle re-encryption.

Reply:
Loading the library shouldn't make us OpenShift only though